packages/xsnap/package.json

```diff
@@ -37,6 +37,7 @@
   },
   "devDependencies": {
     "@endo/base64": "^0.2.8",
+    "@rollup/plugin-commonjs": "^21.0.1",
```

Comment:
Ah, good catch!

Reply:
The sdk wouldn't build without this after removing the old wallet, not sure why /shrug